examples/get-started/first_figure.py

```diff
@@ -0,0 +1,158 @@
+# To add to a plot object (``fig`` in this example), the PyGMT module is used
+# as a method on the class. This example will use the module ``coast``, which
+# can be used to create a map without any other modules or external data. The
+# ``coast`` module plots the coastlines, borders, and bodies of water using a
+# database that is included in GMT.
+#
+# First, a region for the figure must be selected. This example will plot some
+# of the coast of Maine in the northeastern US. A Python list can be passed to
+# the ``region`` argument with the minimum and maximum X-values (longitude)
+# and the minimum and maximum Y-values (latitude). For this example, the
+# minimum (bottom left) coordinates are N43.75 W69 and the maximum (top right)
+# coordinates are N44.75 W68. Negative values can be passed for latitudes in
+# the southern hemisphere or longitudes in the western hemisphere.
+#
+# In addition to the region, an argument needs to be passed to ``coast`` to
+# tell it what to plot. In this example, ``coast`` will be told to plot the
+# shorelines by passing the Boolean value ``True`` to the ``shorelines``
+# parameter. The ``shorelines`` parameter has other options for finer control,
+# but setting it to ``True`` uses its default values.
+
+fig.coast(region=[-69, -68, 43.75, 44.75], shorelines=True)
+
+###############################################################################
+# To see the figure, call :meth:`pygmt.Figure.show`. If you are using a Python
+# script instead of a notebook, use ``fig.show(method="external)`` to display
+# the figure.
+
+fig.show()
+
+###############################################################################
+# Color the land and water
+# ------------------------
+#
+# This figure plots all of the coastlines in the given region, but it does not
+# indicate where the land and water are. Color values can be passed to ``land``
+# and ``water`` to set the colors on the figure.
+#
+# When plotting colors in PyGMT, there are multiple
+# :gmt-docs:`color codes <gmtcolors.html>`, that can be used. This includes
+# standard GMT color names (like ``skyblue``), R/G/B levels (like ``0/0/255``),
+# a hex value (like ``#333333``), and a graylevel (like ``50``). For this
+# example, GMT color names are used.
+
+fig = pygmt.Figure()
+fig.coast(
+    region=[-69, -68, 43.75, 44.75],
+    shorelines=True,
+    land="lightgreen",
+    water="lightblue",
+)
+fig.show()
+
+###############################################################################
+# Set the projection
+# ------------------
+#
+# This figure now has its colors set, but there is not projection or size
+# set for the map. Both of these values are set using the ``projection``
+# parameter.
+#
+# The appropriate projection varies for the type of map. The available
+# projections are explained in the "Projections" gallery. For this example,
+# the Mercator projection is set using "M". The width of the figure will be
```

Comment:
```suggestion
# the Mercator projection is set using ``"M"``. The width of the figure will be
```
These changes are suggested for consistency with earlier in the tutorial, when the
arguments were formatted as code.